web-console/knikubevirt/Create-vm/step-4-advanced/wizard-advanced.md

```diff
@@ -0,0 +1,30 @@
+# Create A Virtual Machine
+
+## Step 4: Advanced
+
+The “Advanced” setting step of the create VM wizard includes:
+
+- Cloud-Init / Sysprep - determines the contents of these settings based on the OS that is selected. (The difference between the cloud-init and the sys-prep is basically the difference between windows and other OSs)
+- Virtual hardware
+
+### Cloud-Init / Sysprep
+
+The user chooses the "Use cloud-init/Sysprep" checkbox at the top of the form and sees the settings.
+
+- SSH Authorized Keys: Enter a comma-separated list of public SSH keys that can be used for authentication.
+- NIC: Select the NIC that should communicate with the DNS servers.
+- DNS Servers: Enter a comma-separated list of DNS server IP addresses.
+- DNS Search Domains: Enter a comma-separated list of DNS search domain IP addresses.
+
+![empty form](img/form-empty.png)
+
+The user checks the “Use Cloud-Init/Sysprep” checkbox in order to enable and fill out the fields below.
+![user fills form](img/ssh-keys.png)
```

Comment:
Should we split out Custom Script into it's own step? I think it's actually separate from cloud-init/sysprep, but maybe something to dig into more.

Reply:
The custom script is actually not a separate step after, it is one or the other. E.g. you can either configure cloud-init using the form or using a custom script. But you can not mix them together.

Reply:
Ah hah...makes sense! I think we could clear this design up a little bit then...there is a choice between the form and custom script (maybe radio buttons since you can not do both?) If custom script is chosen then the user would really only fill out the custom script field.

WDYT? @yfrimanm @matthewcarleton

Reply:
+1 a radio button set that has "Custom Script" or "form"? With form as the default.

Reply:
Haha yeah I'm trying to think of a good way to distinguish the two...

"Cloud Init - Form"
"Cloud Init - Custom Script"

maybe?

Reply:
or maybe something like 
<img width="262" alt="Screen Shot 2019-08-30 at 12 28 15 PM" src="https://user-images.githubusercontent.com/287880/64032849-b1459f00-cb21-11e9-9198-f0bb3cd688ac.png">

Reply:
Looks nice and clear to me!

Reply:
+1